docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -0,0 +1,240 @@
+argument is no longer available because `pprof` information is now always
+provided. For more information, refer to {agent-pull}1140[#1140].
+
+*Impact* +
+Remove the `--pprof` argument from any scripts or commands you use.
+====
+
+[discrete]
+[[known-issues-8.6.0]]
+=== Known issues
+
+[discrete]
+[[known-issue-issue-2066]]
+.Osquery live query results can take up to five minutes to show up in {kib}
+[%collapsible]
+====
+*Details* +
+A known issue in {agent} may prevent live query results from being available
+in the {kib} UI even though the results have been successfully sent to {es}. 
+For more information, refer to {agent-issue}2066[#2066].
+
+*Impact* +
+Be aware that the live query results shown in {kib} may be delayed by up to 5 minutes.
+====
+
+[discrete]
+[[new-features-8.6.0]]
+=== New features
+
+The 8.6.0 release adds the following new and notable features.
+
+{fleet}::
+* Differentiate kubernetes integration multipage experience {kibana-pull}145224[#145224]
+* Add prerelease toggle to Integrations list {kibana-pull}143853[#143853]
+* Add link to allow users to skip multistep add integration workflow {kibana-pull}143279[#143279]
+* Add new action for single agents to request diagnostics {kibana-pull}142369[#142369]
+
+{agent}::
+* Upgrade Node to version 18.12.0 {agent-pull}1657[#1657] 
+* Add experimental support for running the elastic-agent-shipper {agent-pull}1527[#1527] {agent-issue}219[#219]
+* Collect logs from sub-processes via stdout and stderr and write them to a single, unified Elastic Agent log file {agent-pull}1702[#1702] {agent-issue}221[#221]
+* Remove inputs when all streams are removed {agent-pull}1869[#1869] {agent-issue}1868[#1868]
+* No longer restart {agent} on log level change {agent-pull}1914[#1914] {agent-issue}1896[#1896]
+* Add `inspect components` command to inspect the computed components/units model of the current configuration (for example, `elastic-agent inspect components`) {agent-pull}1701[#1701] {agent-issue}836[#836]
+* Add support for the Common Expression Language (CEL) {filebeat} input type {agent-pull}1719[#1719]
+* Only support {es} as an output for the beta synthetics integration {agent-pull}1491[#1491]
+
+[discrete]
+[[enhancements-8.6.0]]
+=== Enhancements
+
+{fleet}::
+* Add `?full` option to get package info endpoint to return all package fields {kibana-pull}144343[#144343]
+
+{agent}::
+* Health Status: {agent} now indicates detailed status information for each sub-process and input type {fleet-server-pull}1747[#1747] {agent-issue}100[#100]
+
+[discrete]
+[[bug-fixes-8.6.0]]
+=== Bug fixes
+
+{fleet}::
+* Only show {fleet}-managed data streams on data streams list page {kibana-pull}143300[#143300]
+* Fix synchronization bug in {fleet-server} that can lead to {es} being flooded by requests to `/.fleet-actions/_fleet/_fleet_search` {fleet-server-pull}2205[#2206].
+
+{agent}::
+* {agent} uses local port when running {fleet-server} {agent-pull}1867[#1867]
```

Comment:
```suggestion
* {agent} now uses the locally bound port (8221) when running {fleet-server} instead of the external port (8220 {agent-pull}1867[#1867]
```